ibr/png-parser/CreateIBR.py

```diff
@@ -0,0 +1,34 @@
+import ibr_pb2
+import sys
+from boundary_extraction.boundary_extraction import extract_boundary
+from imagedata_creation.imagedata_creation import get_image_data
+from utils import *
```

Comment:
nit: good python style is to separate imports into 1) standard libraries, 2) third party libraries, and 3) local modules

https://stackoverflow.com/a/20763446

i.e.
```
import sys

import ibr_pb2

from boundary_extraction.boundary_extraction import extract_boundary
from imagedata_creation.imagedata_creation import get_image_data
from utils import *
```